### PR TITLE
Update cache clearing documentation

### PR DESCRIPTION
**Pull request by designatednerd**
Also added a bit of documentation for a couple undocumented methods. 

Gonna hold off on updating the generated docs until release to avoid confusion.

### CODE REVIEW

File: Sources/Apollo/ApolloStore.swift [hunk 1]
```diff
@@ -44,6 +47,9 @@ public final class ApolloStore {
     }
   }
 
+  /// Clears ths instance of the cache. Note that a cache can be shared across multiple `ApolloClient` objects, so clearing that underlying cache will clear it for all clients.
```
Comment (AnthonyMDev) [inline Sources/Apollo/ApolloStore.swift]: Typo :)
```suggestion
  /// Clears the instance of the cache. Note that a cache can be shared across multiple `ApolloClient` objects, so clearing that underlying cache will clear it for all clients.
```

Comment (designatednerd) [reply Sources/Apollo/ApolloStore.swift]: English may be my first language but typo is clearly my second 🤦‍♀️

**Comment by designatednerd** [comment]
@AnthonyMDev We're always happy to accept contributions, especially since we're now using the inline docs to generate API documentation on our website 😁

**Comment by AnthonyMDev** [comment]
> @AnthonyMDev We're always happy to accept contributions, especially since we're now using the inline docs to generate API documentation on our website 😁

Absolutely! As soon as my company finished approving my open source contribution request I'll be on it! I think that should be done tomorrow!

**Pull request merged**